### DISCUSSION TITLE
Custom template and images for search verticals

### DISCUSSION THREAD

**Discussion by VerlindenEAM** [microsoft-search/pnp-modern-search#3037] [Ideas]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Possibility to use a custom verticals template for custom icons/images in the search verticals (same as for filters).
This way we could add a custom icon to each vertical.

**Describe alternatives you've considered**
Alternative an option to select a custom image/icon url instead of Fabric UI icon.

**Comment by FranckyC** [comment]
HI @VerlindenEAM, thank you for your feedback. I'll tag it as an ehancement.

**Reply by vishalgoyal2472** [reply]
HI @FranckyC 
Is this feature added to newer version of PnP?
